pmd-core/src/test/java/net/sourceforge/pmd/cpd/CPDCommandLineInterfaceTest.java

```diff
@@ -80,12 +85,29 @@ public void testDeprecatedOptionsWarning() throws IOException {
 
         CPD.StatusCode statusCode = CPD.runCpd("--minimum-tokens", "340", "--language", "java", "--filelist",
                 filelist.toAbsolutePath().toString(), "--format", "xml", "-failOnViolation", "true");
-        Assert.assertEquals(CPD.StatusCode.OK, statusCode);
-        Assert.assertEquals("<?xml version=\"1.0\" encoding=\"UTF-8\"?>" + "\n" + "<pmd-cpd>" + "\n" + expectedFilesXml + "</pmd-cpd>", log.getLog());
+        assertEquals(CPD.StatusCode.OK, statusCode);
+        assertEquals("<?xml version=\"1.0\" encoding=\"UTF-8\"?>" + "\n" + "<pmd-cpd>" + "\n" + expectedFilesXml + "</pmd-cpd>", log.getLog());
         assertTrue(loggingRule.getLog().contains("Some deprecated options were used on the command-line, including -failOnViolation"));
         assertTrue(loggingRule.getLog().contains("Consider replacing it with --fail-on-violation"));
         // only one parameter is logged
         assertFalse(loggingRule.getLog().contains("Some deprecated options were used on the command-line, including --filelist"));
         assertFalse(loggingRule.getLog().contains("Consider replacing it with --file-list"));
     }
+
+    @Test
+    public void testDebugLogging() {
+        CPD.StatusCode statusCode = CPD.runCpd("--minimum-tokens", "340", "--language", "java", "--files",
+                SRC_DIR, "--debug");
+        assertEquals(CPD.StatusCode.OK, statusCode);
+        assertTrue(errLog.getLog().contains("Tokenizing ")); // this is a debug logging
```

Comment:
Please use `MatcherAssert.assertThat` and not assertTrue (other tests here don't, but assertTrue gives terrible error messages...)
```suggestion
        assertThat(errLog.getLog(), containsString("Tokenizing ")); // this is a debug logging
```